src/ert/scheduler/lsf_driver.py

```diff
@@ -508,3 +509,30 @@ def _build_resource_requirement_arg(self) -> List[str]:
 
     async def finish(self) -> None:
         pass
+
+    def read_output_files(
```

Comment:
I think "output" is too ambigous here. Include stdout and stderr in function name.

Reply:
`read_stdout_and_stderr_files`